docs/release-notes.rst

```diff
@@ -36,6 +36,10 @@ Changes and New Features
 
    * - Version
      - Description
+   * - 24.10.0
+     - | - Added support for NVIDIA NIC Configuration Operator deployment.
+       | - Added support for Support Single-Node OpenShift (SNO).
+       | - NVIDIA Network Operator Helm chart does not deploy NicClusterPolicy CRD anymore.
```

Comment:
WDYT
- NVIDIA Network Operator Helm chart does not create NicClusterPolicy CR anymore.

Reply:
I like this proposal!